src/pipeline/activator/aviris-l2/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM continuumio/miniconda3:4.8.2 AS app
+
+RUN conda install --yes python=3.8
+COPY ./requirements-conda.txt /tmp/requirements-conda.txt
+RUN conda install --yes --channel conda-forge \
+  --file /tmp/requirements-conda.txt
+
+WORKDIR /workspace
```

Comment:
> I'd consider making the working directory /usr/local/src

Makes sense, I'll update to match existing project conventions. The `/workspace` name was an artifact of vscode dev containers.

> The pip install invocation could go above the COPY requirements.txt instruction

I don't think we can do this here and I don't think we want to. Rather, if the conda layer is invalidated and rebuilt we actually want to also reinstall our pip dependencies. [conda recommends](https://www.anaconda.com/blog/using-pip-in-a-conda-environment):

> Only after conda has been used to install as many packages as possible should pip be used to install any remaining software. If modifications are needed to the environment, it is best to create a new environment rather than running conda after pip

Otherwise, great suggestions, I'll apply these changes to both conda dockerfiles.